<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KasaMasaDialog</class>
 <widget class="QDialog" name="KasaMasaDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>566</width>
    <height>462</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="title">
        <string>Kasalar</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Baƒülƒ± Depo</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_BagliDepo"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QListView" name="listView_kasaList"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="pushButton_deleteKasa">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Sil‚ùå</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_refreshKasaList">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>üîÑ</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_addNewKasa">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Kasa‚ûï</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="title">
        <string>Masalar</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QListView" name="listView_MasaList"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="pushButton_deleteMasa">
            <property name="text">
             <string>Sil‚ùå</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_refreshMasaList">
            <property name="text">
             <string>üîÑ</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_addNewMasa">
            <property name="text">
             <string>Masa‚ûï</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_Kapat">
     <property name="text">
      <string>Kapat</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
